app/harvesters/uva_harvester.rb

```diff
@@ -0,0 +1,157 @@
+##
+# A harvester implementation for UVA
+#
+class UVAHarvester
+  include Krikri::Harvester
+
+  DEFAULT_THREAD_COUNT = 10
+  DEFAULT_HARVEST_NAME = 'virginia'
+  DEFAULT_MAX_RECORDS = 0
+
+  ##
+  # Initialize, and set default options as appropriate.
+  #
+  # @param opts [Hash] a hash of options as defined by {.expected_opts}
+  #
+  # @example
+  #    Typical instantiation, good for most cases:
+  #
+  #      UVAHarvester.new(uri: 'http://example.edu/fedora/...')
+  #
+  # Accepts options passed as `:uva => opts`
+  #
+  # Options allowed are:
+  #
+  #   - threads:     The number of records to fetch asynchronously
+  #                  in a batch (default: 10)
+  #   - name:        See Krikri::Harvester#initialize.
+  #                  Defaults to "virginia"
+  #   - max_records: The maximum number of records to harvest
+  #                  0 means no limit (default 0)
+  #
+  def initialize(opts = {})
+    @opts = opts.fetch(:uva, {})
+    super
+
+    @opts[:threads] ||= DEFAULT_THREAD_COUNT
+    @opts[:name] ||= DEFAULT_HARVEST_NAME
+    @opts[:max_records] ||= DEFAULT_MAX_RECORDS
+
+    @http = Krikri::AsyncUriGetter.new
+  end
+
+  ##
+  # @see Krikri::Harvester.expected_opts
+  def self.expected_opts
+    {
+      key: :uva,
+      opts: {
+        threads:  { type: :integer, required: false },
+        name:  { type: :string, required: false },
```

Comment:
`:name` should not be included in `#expected_opts` (see [here](https://github.com/dpla/KriKri/blob/develop/lib/krikri/harvester.rb#L150-L181))